speech/google/cloud/speech/client.py

```diff
@@ -15,18 +15,31 @@
 """Basic client for Google Cloud Speech API."""
 
 from base64 import b64encode
+import os
 
+from google.cloud.client import JSONClient
 from google.cloud._helpers import _to_bytes
 from google.cloud._helpers import _bytes_to_unicode
-from google.cloud import client as client_module
+from google.cloud.environment_vars import DISABLE_GRPC
 from google.cloud.speech.connection import Connection
 from google.cloud.speech.encoding import Encoding
 from google.cloud.speech.operation import Operation
 from google.cloud.speech.sample import Sample
 from google.cloud.speech.transcript import Transcript
 
+try:
+    from google.cloud.speech._gax import GAPICSpeechAPI
+except ImportError:  # pragma: NO COVER
+    _HAVE_GAX = False
+else:
+    _HAVE_GAX = True
 
-class Client(client_module.Client):
+
+_DISABLE_GAX = os.getenv(DISABLE_GRPC, False)
+_USE_GAX = _HAVE_GAX and not _DISABLE_GAX
+
+
+class Client(JSONClient):
```

Comment:
```python
        super(Client, self).__init__(project=project, credentials=credentials,
                                     http=http)
```

I remember that `super()` is kind of weird but I would think that it would pick up the inheritance of `_ClientProjectMixin`.